test/end-to-end/core.sh

```diff
@@ -184,6 +184,11 @@ os::cmd::expect_success_and_text "cat '${LOG_DIR}/kubectl-with-token.log'" 'Usin
 os::cmd::expect_success_and_text "cat '${LOG_DIR}/kubectl-with-token.log'" 'kubectl-with-token'
 
 echo "[INFO] Testing deployment logs and failing pre and mid hooks ..."
+# test hook selectors
+oc create -f test/fixtures/hooks-dc.yaml
```

Comment:
Also since we don't use this deployment anymore, delete it after you are done testing.